runtime/src/main/java/org/corfudb/runtime/collections/TxBuilder.java

```diff
@@ -99,6 +100,19 @@ TxBuilder update(@Nonnull final String tableName,
         return this;
     }
 
+    /**
+     *
+     * @param streamId      The stream to perform the update.
+     * @param updateEntry   The entry to be applied to the stream.
+     * @return TxBuilder Instance
+     */
+    public TxBuilder logUpdate(UUID streamId, SMREntry updateEntry) {
```

Comment:
UUID from caller's perspective makes very little sense.
Please accept a tableName instead and generate the UUID using 
UUID.nameUUIDFromBytes(TableRegistry.getFullyQualifiedTableName(namespace, tableName))